File: CS/10_Conversion/ConvertODPtoPDF.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;

namespace ConvertODPtoPDF
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {

            Presentation presentation = new Presentation();

            //Load ODP file from disk
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\toPdf.odp",FileFormat.ODP);

            String result = "ConvertODPtoPDF_result.pdf";

            //Save to file.
            presentation.SaveToFile(result, FileFormat.PDF);

            //Launch the PowerPoint file.
            DocumentViewer(result);
        }

        private void DocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB/10_Conversion/ConvertODPtoPDF.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation

Namespace ConvertODPtoPDF
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()

		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click

			Dim presentation As New Presentation()

			'Load ODP file from disk
			presentation.LoadFromFile("..\..\..\..\..\..\Data\toPdf.odp",FileFormat.ODP)

			Dim result As String = "ConvertODPtoPDF_result.pdf"

			'Save to file.
			presentation.SaveToFile(result, FileFormat.PDF)

			'Launch the PowerPoint file.
			DocumentViewer(result)
		End Sub

		Private Sub DocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace